src/components/SelectArticle.jsx

```diff
@@ -24,6 +36,23 @@ const SelectArticle = () => {
       <p>Votes: {article.votes}</p>
       <img src={article.article_img_url} alt={article.title} />
       <p>{article.body}</p>
+      <ul className="comment-ul"> Comments
+        {Object.values(commentsById).map((comment) => {
+        return(
+            <li className="comment-li" key={comment.comment_id}> 
+            By: {comment.author} 
+            <br />
+            {comment.body} 
+            <br />
+            Created: {comment.created_at}
+            <br />
+            Votes: {comment.votes}
+            </li>
+            
+
+        )
+    })}
+    </ul>
```

Comment:
This could be extracted into a separate component eg Comments.jsx. ALong with the api call. It would be good to include loading state as well whilst the comments are being fetched.